cfme/web_ui/__init__.py

```diff
@@ -1280,6 +1280,7 @@ def _fill_form_list(form, values, action=None, action_always=False):
 
     """
     logger.info('Beginning to fill in form...')
+    sel.wait_for_ajax()
```

Comment:
Well, sometimes even ajax is not enough. I remember I had to put a special waits for element in some form navigation steps because wait_for_ajax just did not catch it. It should be possible to write a JS snippet that returns if the element is in focus and that could be waited for so nothing interferes with the filling process...